cli/flags.rs

```diff
@@ -1232,17 +1233,15 @@ fn permission_args_parse(flags: &mut Flags, matches: &clap::ArgMatches) {
 }
 
 // TODO(ry) move this to utility module and add test.
-/// Strips fragment part of URL. Panics on bad URL.
+/// Strips fragment part of URL.
 pub fn resolve_urls(urls: Vec<String>) -> Vec<String> {
```

Comment:
This method should rather return `Result<Vec<String>, Error>`; otherwise there's no way to write unit test for failing URL - process will exit instead of erroring and running other tests

Reply:
I agree. We shouldn't make it more ergonomic to `process::exit(1)`, instead we should be removing those.

@kevinkassimo I appreciate the patch as always - but I'd rather the ugly panic than introducing this exit-macro. Closing without merge.